docu/files/35_wherenext.html.eco

```diff
@@ -6,4 +6,8 @@ title: 'Where Do I Go Next?'
 <p>If you are not a developer, then the&nbsp;<a href="https://knowledge.yaas.io/">Knowledge Hub</a>&nbsp;is the right place for you. As you have already seen in this section, the training provides you with free online training on various YaaS topics.&nbsp;We provide you with the&nbsp;guides that help you configure and use your packages efficiently. Also,&nbsp;you can get recommendations, examples of common pitfalls and best approaches, news, and more.</p>
 <p>Finally, if you are interested in using YaaS to build on your&nbsp;existing SAP portfolio, you can use guides and articles in&nbsp;the Knowledge Hub or look on the DevPortal for the technical details.</p>
+<p>Beside resources available on the Knowledge Hub and Dev Portal, you might be interested in the training offerings from SAP Hybris Education:</p>
+<li>For developers, SAP Hybris Education offers&nbsp;<a href="https://training.sap.com/shop/course/hy950-sap-hybris-as-a-service-yaas-developer-classroom-002-g-en/" rel="nofollow">SAP Hybris as a Service Developer</a>&nbsp;classroom training. This training is designed for Java J2EE developers and consists of 20% lectures and 80% hands-on exercises. In this training course, participants will learn how to develop a microservice and a package using YaaS SDK.</li>
+<li>For Solution Architect, there is SAP Hybris Solution Architect: SAP Hybris Commerce and YaaS Online training.&nbsp;This training is designed to provide participants with a thorough understanding of how to connect applications in the SAP Hybris Portfolio to share data and services. The focus of this training is on integrating a front office architecture consisting of the SAP Hybris Commerce Electronics storefront and SAP Hybris Loyalty.&nbsp;This course&nbsp;provides a high-level introduction to the functionality and technical background of SAP Hybris Commerce, SAP Hybris Data Hub and SAP Hybris Loyalty.</li>
+<li>For decision makers, functional&nbsp;analysts and leads, SAP Hybris Education&nbsp;offered free of charge SAP Hybris profile Essentials training.&nbsp;The course introduces you to the features and business benefits of SAP Hybris Profile by taking you through the customer profile's journey:&nbsp;how SAP Hybris Profile creates, enriches, maintains and updates a customer profile. You can watch the demos that present available SAP Hybris Profile integrations with other SAP Hybris solutions.&nbsp;</li>
```

Comment:
<li>For decision makers, functional&nbsp;analysts and leads, SAP Hybris Education&nbsp;offers, free of charge, SAP Hybris Profile Essentials training.&nbsp;The course introduces you to the features and business benefits of SAP Hybris Profile by taking you through the customer profile's journey:&nbsp;how SAP Hybris Profile creates, enriches, maintains, and updates a customer profile. You can watch demos that present available SAP Hybris Profile integrations with other SAP Hybris solutions.&nbsp;</li>